kolibri/plugins/coach/assets/src/routes/index.js

```diff
@@ -13,6 +13,18 @@ import reportRoutes from './reportRoutes';
 import planRoutes from './planRoutes';
 import { classIdParamRequiredGuard } from './utils';
 
+async function showHomePage(toRoute) {
+  console.log('Inside showHomePage');
```

Comment:
Please remove `console.log`. I am not sure why lint check didn't detect it.

Reply:
I have removed it.